src/main/java/com/smalaca/messagesender/domain/UserFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.smalaca.messagesender.domain;
+
+import com.smalaca.messagesender.service.UserDto;
+
+public class UserFactory {
```

Comment:
For the Factory + Builder + private constructor
![image](https://user-images.githubusercontent.com/4753177/36944082-5596cf82-1f95-11e8-9c9f-a9fe4753184a.png)